setup/general/kuksa.val2iotea/config/config.json

```diff
@@ -1,8 +1,19 @@
 {
-    "loglevel": "DEBUG",
-    "mqtt": {
-        "connectionString": "mqtt://localhost:1883",
-        "ns": "iotea/"
+    "loglevel": "INFO",
+    "protocolGateway": {
+        "adapters": [
+            {
+                "platform": true,
+                "module": {
+                    "name": "./util/mqttClient",
+                    "class": "MqttProtocolAdapter"
+                },
+                "config": {
+                    "topicNamespace": "iotea/",
+                    "brokerUrl": "mqtt://mosquitto-local:1883"
```

Comment:
Why mosquitt-local and not mosquitto? The DNS name for the service is set in the `docker-compose.stack.yml`

Reply:
Updated.